### PR TITLE
[Durable Objects] minor DX updates

### PR DESCRIPTION
**Pull request by rita3ko**

### CODE REVIEW

File: content/durable-objects/examples/durable-object-location-example.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 type: example
-summary: Create a simple Durable Object.
+summary: Create a Durable Object that keeps track of the location of the durable objects
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/examples/durable-object-location-example.md]:
```suggestion
summary: Create a Durable Object that keeps track of the location of Durable Objects.
```

File: content/durable-objects/get-started.md [hunk 2]
```diff
@@ -192,9 +192,12 @@ Refer to [Durable Objects migrations](/durable-objects/learning/durable-objects-
 To test your Durable Object locally, run [`wrangler dev`](/workers/wrangler/commands/#dev):
 
 ```sh
-$ wrangler dev
+$ npx wrangler dev
 ```
 
+In your console you should see: `Hello world` which is bring returned by the Durable Objects.
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/get-started.md]:
```suggestion
In your console, you should see a`Hello world` string returned by the Durable Object.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/_redirects [hunk 3]
```diff
@@ -257,6 +257,9 @@
 /support/dns/troubleshooting/my-dns-doesnt-work/ /dns/troubleshooting/dns-issues/ 301
 /support/dns/troubleshooting/why-was-my-domain-deleted-from-cloudflare/ /dns/zone-setups/troubleshooting/domain-deleted/ 301
 
+# durable objects
+/durable-object-example/ /durable-object-location-example/ 301
```
Comment (pedrosousa) [inline content/_redirects]:
```suggestion
/durable-objects/examples/durable-object-example/ /durable-objects/examples/durable-object-location-example/ 301
```

**Comment by Maddy-Cloudflare** [comment]
@rita3ko  do you have permissions to merge this?

File: content/durable-objects/examples/durable-object-location-example.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 ---
 type: example
-summary: Create a simple Durable Object.
+summary: Create a Durable Object that keeps track of the location of Durable Objects.
 tags:
   - Durable Objects
 pcx_content_type: configuration
-title: Durable Object example
+title: Durable Object Location example
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/examples/durable-object-location-example.md]:
```suggestion
title: Durable Object location example
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**